src/snowflake/snowpark/modin/plugin/io/snow_io.py

```diff
@@ -658,19 +658,20 @@ def read_stata(
         pass  # pragma: no cover
 
     @classmethod
-    @pandas_module_level_function_not_implemented()
-    def read_sas(
-        cls,
-        filepath_or_buffer,
-        *,
-        format=None,
-        index=None,
-        encoding=None,
-        chunksize=None,
-        iterator=False,
-        **kwargs,
-    ):
-        pass  # pragma: no cover
+    def read_sas(cls, **kwargs):  # noqa: PR01
+        """
+        Read SAS files stored as either XPORT or SAS7BDAT format files into a query compiler.
+
+        Snowpark pandas has a slightly different error message from the upstream modin version.
+        """
+        intermediate = pandas.read_sas(**kwargs)
+        if isinstance(intermediate, (OrderedDict, dict)):  # pragma: no cover
```

Comment:
qq: why we want to exclude this from test coverage?

Reply:
This check has been removed now. It was inherited from the implementation of `read_excel`. However, it does not apply here - because `read_excel` may return a `dict` while `read_sas` does not.